client/my-sites/email/email-management/email-home.jsx

```diff
@@ -110,6 +111,7 @@ class EmailManagementHome extends React.Component {
 					domains={ domainsWithEmail }
 					selectedSiteSlug={ selectedSite.slug }
 					currentRoute={ currentRoute }
+					siteId={ selectedSiteId }
```

Comment:
I think it would be better to rename the `siteId` prop to `selectedSiteId` to be more consistent with the `selectedSiteSlug` prop, and also with other components under `/client/my-sites/email`. Note I would keep using `siteId` in the `useEmailsQuery` function though (since it's in another folder).